### PR TITLE
Remove lodash omit

### PR DESCRIPTION
**Pull request by chrisbolin**
see https://github.com/FormidableLabs/victory/issues/956 for background and process.

this is all that will need to change in `victory-core`. i won't merge this until I start testing with `victory-chart`

### CODE REVIEW

File: src/victory-util/helpers.js [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
 
 export default {
+  /**
+   * creates an object with some keys excluded
+   * replacement for lodash.omit for performance. does not mimick the entire lodash.omit api
+   * @param {Object} originalObject: created object will be based on this object
+   * @param {Array<String>} keys: an array of keys to omit from the new object
+   * @returns {Object} new object with same properties as originalObject
+   */
+  omit(originalObject, keys = []) {
```
Comment (ryan-roemer) [inline src/victory-util/helpers.js]: Can you refactor this to **not** be part of the `default` export? 

We've got tickets already to unwind shoving everything in this mega object.

Then just switch to `import { omit } from "../victory-util/helpers";` and we're off!

File: src/victory-util/helpers.js [hunk 2]
```diff
@@ -148,7 +170,7 @@ export default {
 
   modifyProps(props, fallbackProps, role) {
     const theme = props.theme && props.theme[role] ? props.theme[role] : {};
-    const themeProps = omit(theme, ["style"]);
+    const themeProps = this.omit(theme, ["style"]);
```
Comment (ryan-roemer) [inline src/victory-util/helpers.js]: Can just revert this change when `omit` is a named export in this same file.

**Review by ryan-roemer — CHANGES_REQUESTED** [review]
Looking good!

Have a refactor, then ping for re-review.

File: src/victory-util/helpers.js (hunk 1 above)
Comment (chrisbolin) [reply src/victory-util/helpers.js]: oh nice! yeah i'd prefer that anyway :)

**Comment by chrisbolin** [comment]
@ryan-roemer - i made those changes!

**Review by ryan-roemer — APPROVED** [review]
💯

**Review by boygirl — APPROVED** [review]
LGTM! Thanks Chris
